Soccar [UI]/src/nl/soccar/ui/fx/controller/GameViewFXMLController.java

```diff
@@ -114,7 +110,19 @@ public void spawnEntity(Entity entity) {
      * @param paused Game is paused, not null.
      */
     public void setPaused(boolean paused) {
-        engine.getGame().setPaused(paused);
+        gameCanvas.getGameEngine().getGame().setPaused(paused);
+    }
+
+    public void stop() {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)